schema/meta-schema.json

```
@@ -225,21 +231,31 @@
       ],
       "properties": {
         "deprecatedVersion": {
+          "title": "Deprecated version",
+          "description": "The OCDS version in which this field was first deprecated.",
```

```suggestion
          "description": "The OCDS version in which the field was first deprecated.",
```

see previous comment